testsuite/tests/typing-safe-linking/Makefile

```diff
@@ -3,6 +3,13 @@
 SOURCES = a.ml b_bad.ml
 OBJECTS = $(SOURCES:%.ml=%.cmo)
 
+default:
+	@if $(SAFE_STRING) ; then \
+	  echo " ... testing 'b_bad.ml' => skipped" ; \
```

Comment:
This is precisely the point: `-safe-string` should not discard the warning, as we could create  a `Y` of type `string t` in another module.